DEVELOPMENT.md

```diff
@@ -0,0 +1,351 @@
+## Getting Started
+
+In order to get started with the project, you will need to have the following
+prerequisites installed on your machine.
+
+### Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [Rust](https://www.rust-lang.org/tools/install)
+- [Node.js](https://nodejs.org/en/)
+
+### Installation
+
+1. Fork the repository (optional)
+2. Clone the repository
+   ```shell
+   git clone git@github.com:bencherdev/bencher.git
+    ```
+3. Checkout the `develop` branch
+   ```shell
+   git checkout devel
+   ```
+4. Build and run the project
+   ```shell
+   docker/run.sh
+   ```
+   or run directly with docker compose using the following command:
+
+   **X64**
+   ```shell
+   ARCH=amd64 docker compose up --file docker/docker-compose.yml --build
+   ```
+    **ARM64**
+   ```shell
+   ARCH=arm64 docker compose up --file docker/docker-compose.yml --build
+   ```
+### Accessing the application
+
+Once the application is running, you can access it by visiting
+[http://localhost:3000](http://localhost:3000) in your browser.
+
+1. Sign up for an account by entering your name and email address.
+2. Click on the "Confirm Email" link that is provided in the docker compose logs.
```

Comment:
This would need to be updated to just the API server logs, as we won't be using `docker compose` for development.